variables.tf

```
@@ -72,4 +72,10 @@ variable "shareable_link_enabled" {
   description = "Option in PREVIEW!!! Allow users connect to a target resource using Azure Bastion without accessing the Azure portal, only available in the Standard tier"
   type        = bool
   default     = null
+}
+
+variable "tags" {
+  type        = map(string)
+  description = "If specified, will set tags for all resources deployed by this module where supported."
+  default     = {}

```

```suggestion
  default     = null
```
It's usually better to use null, as tf treats that as if the property wasn't used in the first place. It doesn't really matter for tags though, as the default value for tags is `{}`

---

Applied recommendation.